www/googlemaps-cdv-plugin.js

```diff
@@ -2632,10 +2632,13 @@ function getAllChildren(root) {
     var clickable;
     var style, displayCSS, opacityCSS, visibilityCSS, node, clickableSize;
 
-    var allClickableElements = Array.prototype.slice.call(root.querySelectorAll(':not([data-clickable="false"])'));
-    var clickableElements =  allClickableElements.filter(function(i) {return i != root;});
+    root = document.querySelector('.main-region.foreground');
```

Comment:
a sibling? I'm pretty sure it's an ancestor! (in the gmaps version, not in the leaflet one)